src/schema/v2/conversation/message.ts

```diff
@@ -134,5 +134,13 @@ export const MessageType = new GraphQLObjectType<any, ResolverContext>({
       resolve: ({ metadata }) => isInvoiceMessage(metadata),
     },
     createdAt: date(),
+    to: {
```

Comment:
shouldn't we cover `to` and `cc` in the tests too?

Reply:
I wasn't too sure about what to test. I added `to` and `cc` to the snapshot on https://github.com/artsy/metaphysics/pull/4698/commits/8c2824aa58f963b21a128c8fe93680be53b0337e. Not sure if it makes a huge difference, though. Thoughts?

Reply:
I'm not a huge fan of snapshots but if was already here let's leave it like this, thanks for addressing it!